.prettierrc

```diff
@@ -1,4 +0,0 @@
-{
-  "semi": false,
```

Comment:
Did you delete this file to avoid lint conflicts, If no, What was the reason to delete it?